src/Servers/Kestrel/Core/src/Internal/Infrastructure/KestrelConnection.cs

```diff
@@ -56,7 +56,7 @@ public void TickHeartbeat()
             }
         }
 
-        public abstract BaseConnectionContext GetTransport();
+        public abstract BaseConnectionContext TransportConnection { get; }
```

Comment:
😆

Reply:
What is this java 🙃

Reply:
I almost left a comment on the original PR, but I didn't want to be too nitpicky considering it's internal. I agree this looks better though.